tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/settings_form.py

```diff
@@ -35,18 +36,13 @@ def __init__(self, marionette):
         Base.__init__(self, marionette)
         view = self.marionette.find_element(*self._settings_view_locator)
         Wait(self.marionette).until(lambda m: view.location['y'] == 0)
+        self._order_by_last_name_switch = GaiaBinaryControl(self.marionette, self._order_by_last_name_switch_locator)
 
-    def tap_order_by_last_name(self):
-        last_name = Wait(self.marionette).until(
-            expected.element_present(*self._order_by_last_name_switch_locator))
-        Wait(self.marionette).until(expected.element_displayed(last_name))
-        initial_state = self.is_custom_element_checked(last_name)
-        last_name.tap()
-        self.wait_for_custom_element_checked_state(last_name, checked=not(initial_state))
+    def enable_order_by_last_name(self):
```

Comment:
So I guess what you mean is that you consider all the elements on a page to be internal objects that should not be visible to outside objects?
I guess that if they then did a redesign of the page, where they didn't use a checkbox, then we only have to change do_something_with_object2() as opposed to fix all the instances my_object1.object2.do_something() (I don't know what you mean with this object3 thing).
I'm not sure how much trouble this would be in practice. In general, at most we use these .object2 things like 5 times maybe.